src/index.ts

```diff
@@ -179,6 +205,7 @@ export const runDev = (
     }
 
     if (compiler.tsConfigPath) {
+      requiredFiles[compiler.tsConfigPath] = true
```

Comment:
`tsconfig.json` doesn't get `require`d but we still want to restart if it changes